pkg/tools/doc/testdata/pkg/md/__main__/Container.md

```diff
@@ -0,0 +1,16 @@
+## Schema Container
+
+Container is the common user interface for long-running services.
+
+### Attributes
+
+**name** *required*
+
+`str`
+
+The name of the long-running container.
+
+
+## Source Files
+
+- [__main__.Container](__main__.server.k)
```

Comment:
You are right, I was about to fix that and since you mentioned, I updated the template and the `__main__` package path will be omitted, while other package paths will be resolved to a full path. Here's an example: `pkg/tools/gen/testdata/doc/pkg/md`